pkg/managed/istioconfigmap.go

```diff
@@ -371,6 +371,19 @@ func getComponentScrapeConfigInfoList(mbPair *types.ModelBindingPair, secretList
 		}
 	}
 
+	// Get all generic bindings info
+	for _, component := range mbPair.Model.Spec.GenericComponents {
+		// Get the common binding info
+		componnentBindingInfo, err := getComponentScrapeConfigInfo(mbPair, component.Name, clusterName, GenericName, GenericKeepSourceLabels,
+			strings.Replace(GenericKeepSourceLabelsRegex, ComponentBindingNameHolder, component.Name, -1))
+		if err != nil {
+			return nil, err
+		}
+		if componnentBindingInfo.BindingName != "" {
```

Comment:
component is spelled wrong in several places, please fix them (e.g. componnentBindingInfo)

Reply:
Done